tensorrt_llm/_torch/visual_gen/models/hunyuan_video1_5/pipeline_hunyuan_video1_5.py

```
@@ -0,0 +1,774 @@
+            num_videos_per_prompt=req.params.num_images_per_prompt,
+            frame_rate=req.params.frame_rate,
+        )
+
+    @torch.inference_mode()
+    def forward(
+        self,
+        prompt: Union[str, List[str]],
+        seed: int,
+        height: int,
+        width: int,
+        negative_prompt: Union[str, List[str]] = None,
```

Ruff (0.16.1)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)